src/dynamicprompts/samplers/combinatorial.py

```diff
@@ -149,13 +149,14 @@ def _get_wildcard(
         command: WildcardCommand,
         context: SamplingContext,
     ) -> StringGen:
+        # TODO: doesn't support weights
```

Comment:
I don't think combinatorial needs to honour weights. Weights imply a random distribution

Reply:
Depends! One might want combinatorial/cyclical sampling to repeat weighted wildcards `weight` times.

Reply:
perhaps... 
nice PR btw